libr/core/canal.c

```diff
@@ -2440,7 +2433,7 @@ R_API int r_core_anal_search(RCore *core, ut64 from, ut64 to, ut64 ref) {
 	if (!buf) {
 		return -1;
 	}
-	r_io_use_desc (core->io, core->file->desc);
+	r_io_desc_use (core->io, core->file->desc->fd);
```

Comment:
this function renaming is wrong. because r_io_desc are methods of the RIODesc struct, so the reason for having r_io_use_desc is because thats a method of RIO, ok to change from desc to fd